release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -99,6 +99,13 @@ For more information, see xref:../installing/cluster-capabilities.adoc#cluster-c
 ==== Root volume types parameter for {rh-openstack} is now available
 You can now specify one or more root volume types in {rh-openstack}, by using the `rootVolume.types` parameter. This parameter is available for both control plane and compute machines.
 
+[id="ocp-4-14-quickly-install-cluster-aws-local-zones"]
+==== Install a cluster quickly in AWS Local Zones
+For {product-title} {product-version}, you can quickly install a cluster on Amazon Web Services (AWS) to extend compute nodes to Local Zone locations. By using this installation route, the installation program automatically creates network resources and Local Zone subnets for each Local Zone that you defined in your configuration file. 
```

Comment:
I would remove duplicated LZ naming, and keep more generic (installer will not create only the subnet), adding the configuration introduced. Something like this:
```suggestion
For {product-title} {product-version}, you can quickly install a cluster on Amazon Web Services (AWS) to extend compute nodes to Local Zone locations. By using this installation route, the installation program fully automate the creation of required resources in Local Zones, network and compute, when the zone names are added in the installation configuration file.
```

I think we could explicity share 

 the installation program automatically creates network resources and Local Zone subnets for each Local Zone that you defined in your configuration file.

Reply:
Good point. I updated the suggestion with some slight changes. WDYT?